packages/badge/src/Default/Default.types.ts

```diff
@@ -1,12 +1,8 @@
-import React from 'react';
 import * as bs from 'react-bootstrap';
 
 export interface BadgeDefaultCommons {
```

Comment:
Remember we're intersecting the `BadgeProps`  here `export type BadgeDefaultProps = BadgeDefaultCommons & bs.BadgeProps;` all `BadgeProps` properties are merged into the `BadgeDefaultProps`, which means we don't need to add them to the `BadgeDefaultCommons`.